ArcGISRuntimeSDKQt_QMLSamples/AR/DisplayScenesInTabletopAR/PermissionsHelper.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019 ESRI
+//
+// All rights reserved under the copyright laws of the United States
+// and applicable international laws, treaties, and conventions.
+//
+// You may freely redistribute and use this sample code, with or
+// without modification, provided you include the original copyright
+// notice and use restrictions.
+//
+// See the Sample code usage restrictions document for further information.
+//
+
+#ifndef PERMISSIONSHELPER_H
+#define PERMISSIONSHELPER_H
+
+#include <QObject>
+
+class PermissionsHelper : public QObject
+{  
+  Q_OBJECT
+
+  Q_PROPERTY(bool fileSystemAccessGranted READ fileSystemAccessGranted NOTIFY fileSystemAccessGrantedChanged)
+
+public:
+  explicit PermissionsHelper(QObject* parent = nullptr);
+  ~PermissionsHelper();
+
+  Q_INVOKABLE void requestFilesystemAccess();
+
+  bool fileSystemAccessGranted() const;
+
+signals:
+  void fileSystemAccessGrantedChanged();
+  void requestFilesystemAccessCompleted(); // notifies even if request denied
+
+private:
+  bool fileSystemAccessCheckRequired() const;
```

Comment:
This method is not defined. Please remove it along with the private section. It's not a mistake on your part, it's undefined in the original. I think it's some leftover junk that got missed in the original code review/PR.